src/sentry/integrations/jira_server/integration.py

```diff
@@ -1007,8 +1007,17 @@ def create_issue(self, data, **kwargs):
                 cleaned_data["summary"] = title[:255] if title else None
                 continue
             elif field_name == "labels" and "labels" in data:
-                labels = [label.strip() for label in data["labels"].split(",") if label.strip()]
-                cleaned_data["labels"] = labels
+                raw_labels_input = data["labels"]
+                processed_labels = []
+                if isinstance(raw_labels_input, str):
+                    processed_labels = [
+                        label.strip() for label in raw_labels_input.split(",") if label.strip()
+                    ]
+                elif isinstance(raw_labels_input, list):
+                    processed_labels = [
+                        str(label).strip() for label in raw_labels_input if str(label).strip()
+                    ]
```

Comment:
While this fix technically addresses the linked issue, it sidesteps the source of the problem which is: labels being incorrectly passed as arrays vs. a single string with comma separated entries. My guess is that these alerts are being created via our API, which doesn't have the same validation our UI has for these.